docs/Parameters.md

```diff
@@ -54,8 +54,8 @@ The parameter format is `key1=value1 key2=value2 ... ` . And parameters can be s
   * support multi validation data, separate by `,`
 * `num_iterations`, default=`100`, type=int, alias=`num_iteration`,`num_tree`,`num_trees`,`num_round`,`num_rounds`
   * number of boosting iterations
-  * note: `num_tree` here equal with `num_iterations`. For multi-class, it actually learns `num_class * num_iterations` trees.
-  * note: For python/R package, cannot use this parameters to control number of iterations.
+  * note: For python/R package, **this parameters is ignored**.
```

Comment:
I do agree that there's a `num_boost_round` parameter in Python package. And that it has a default value. (And this value is different for `train` and `cv`, which is a bit strange.) However, I still can pass any positive value for `num_iterations` via `params` dict. The calculation result would be the same, regardless of the value specified. So `num_iterations` is _ignored_ - there's no `lightgbm.basic.LightGBMError: b'Unknown parameter` or warning, or something.

In R package it works just the same, except that it is called `nrounds` here (and the default value is 10 for both `train` and `cv`).

But I do agree that simply saying "it is ignored" might be misleading as well, because it may look like if there's absolutely no way to control it (That would be crazy for such an important parameter, but whatever.). So, maybe the following version would be OK:

> note: For python/R package, **this parameter is ignored**, use `num_boost_round` (Python) or `nrounds` (R) method arguments to specify number of iterations` 

?